TimeEntry: don't post an empty array

chtitux opened the pull request.

Commit:
TimeEntry: don't post an empty array

paulRbr commented on lib/dovico/model/time_entry.rb:
```diff
@@ -54,7 +54,9 @@ def self.search(start_date, end_date)
-      ApiClient.post(URL_PATH, body: api_assignements.to_json)
+      if api_assignements.count > 0
+        ApiClient.post(URL_PATH, body: api_assignements.to_json)
+      end
```
should we output an error message if assignments are empty?

chtitux replied:
No sure, because if happens when the user set no time entries to be generated.

So it's expected for the user to have 0 entries to be posted.

chtitux commented:
@paulRbr I have updated the MR:
- it's now rebased and mergeable
- We return an empty array if no time entries are to be created (the result of this method is expected to be an array with elements created, used by [`lib/dovico/app.rb`](lib/dovico/app.rb)
- Specs have been added to test with no entries to be created
